shuttheboxgame.py

```diff
@@ -14,26 +14,31 @@
 
 class Strategy(object):
     def __init__(self, tiles, roll_sum):
```

Comment:
Do you know about type hints and doc strings? I would recommend adding those to each function

Example:
Type hints: `def __init__(self, tiles: list, roll_sum: int)`
* These show the variable type that is being inputted which makes understanding the code easier

Doc string (provides a brief description of the function and its inputs and outputs)
```def __init__...:
"""
Initialize the strategy object
:param tiles: List of tiles still standing
:param roll_sum: Sum of dice roll to knock down the tiles
"""
```

Reply:
Added! Good call